Update readme.md (#71)
infinityworks/policies: ISO-27001/databreach/readme.md

Infinity Works' Originated Incidents

Any individual who accesses, uses or manages the company’s information is responsible for reporting data breach and information security incidents immediately to Infinity Works' Data Protection Group (data.protection@infinityworks.com) - This should be done by completing a [Data Breach Incident Report Form](https://goo.gl/forms/VfHKbokfpRdGAc9f2).
 
The Data Breach Incident Report Form must include full and accurate details of the incident, when the breach occurred (dates and times), who is reporting it, if the data relates to people, the nature of the information, and how many individuals are involved.  

> The Data Protection Group (which comprises IW's Technical Directors, Operations Directors, Legal Counsel, IT Systems and Group Operations Director) will be automatically informed of the breach through email and Slack.  The breach will be triaged, assigned owners, resolved, reported and escalated as per the diagram below. 

Reviewer: Do we need to specify who is in the group, as this seems likely to go stale? i.e. could we just remove that:
```suggestion
Members of the Data Protection Group will be automatically informed of the breach through email and Slack. The breach will be triaged, assigned owners, resolved, reported and escalated as per the diagram below. 
```
